recipe/meta.yaml

```diff
@@ -20,33 +20,37 @@ requirements:
   host:
     - python
     - pip
-    - setuptools
+    - setuptools >=39.0.0
+    - setuptools_scm >=3.4.3
     - wheel
   run:
+    # - importlib-metadata <4.3  # flake 8 requires an older version of importlib-metadata
     - autopep8 >=1.6.0,<1.7.0
     - flake8 >=4.0.0,<4.1.0
     - jedi >=0.17.2,<0.19.0
-    - mccabe >=0.6.0,<0.7.0
```

Comment:
You can safely use `mccabe >=0.6.0,<0.8.0` and skip `pip check`

Reply:
I thought that we wanted to have `mccabe` as a requirement for `flake 8`